css/clay_css_2_recipes/01_getting_started.md

```diff
@@ -223,53 +233,55 @@ Clay CSS uses `rem` based values instead of `px` for accessibility. This ensures
 
 ## Headers (h1, h2, h3, h4, h5, h6)
 
-Material Design header styles:
+Material Design's header styles are:
 
-```
-h1, h2, h3, h4, h5 {
+```css
+h1,
+h2,
+h3,
+h4,
+h5 {
 	color: #212121;
-	font-family: Roboto,sans-serif;
+	font-family: Roboto, sans-serif;
 	font-weight: $font-weight-normal;
 	line-height: 1.2;
 	margin: 0 0 8px;
 }
 ```
 
-We can change these in Clay CSS with:
+To make the headers look the same in Clay CSS, we need to change:
 
-```
+```scss
 $headings-color: $gray-900;
 $headings-font-weight: 400;
 $headings-margin-bottom: 0.5rem;
 ```
 
-We don't need to change:
+We can keep the following variables set to their default values, since the `family-family` is already set on the body and `$headings-line-height` is already set to 1.2.
```

Comment:
`family-family`? Should that be `font-family`?

Reply:
most definitely, pushed new changes. Thanks @wincent!